kiwi/utils/rpm_database.py

```diff
@@ -133,3 +133,16 @@ def set_database_to_image_path(self):
             self.rpmdb_image.write_config()
             self.rebuild_database()
             self.rpmdb_image.wipe_config()
+
+            root_rpm_host_dbpath = os.path.normpath(
+                os.sep.join([self.root_dir, rpm_host_dbpath])
```

Comment:
Is the use of `os.sep` instead of `os.path.join` intended here? If no there is one detail that needs to be taken into consideration when `rpm_host_dbpath` is an absolute path:

```python
>>> import os
>>> os.path.normpath(os.sep.join(['/var', 'lib/rpm.db']))
'/var/lib/rpm.db'
>>> os.path.normpath(os.sep.join(['/var', '/lib/rpm.db']))
'/var/lib/rpm.db'
>>> os.path.join('/var', 'lib/rpm.db')
'/var/lib/rpm.db'
>>> os.path.join('/var', '/lib/rpm.db')
'/lib/rpm.db'
```

There are situations where if rpm_host_dbpath is absolute you are looking for os.join.path behavior.

Reply:
Nope the use of os.sep is intentional here. That's because I intentionally don't want the intelligence of os.path.join. Your last example demonstrates that. The way it's used in the method
is join(root_dir, some_path). I have to trust the join method to join those two in __any__ case.
Now the os.path.join method is super clever and does not join as I expect it if the argument list of the given join path elements is absolute

```python
>>> os.path.join('/var', '/lib/rpm.db')
'/lib/rpm.db'
```

That would be horrible because the code would then no longer point to the image root but your host system and people will kill me if I remove their host rpm database :)

Reply:
Great, thanks!